addons/web/static/tests/views/fields/statusbar_time_field_tests.js

```diff
@@ -0,0 +1,112 @@
+/** @odoo-module **/
+
+import {click, getFixture} from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+
+let serverData;
+let target;
+
+QUnit.module("Fields", (hooks) => {
+    hooks.beforeEach(() => {
+        target = getFixture();
+        serverData = {
+            models: {
+                partner: {
+                    fields: {
+                        display_name: { string: "Displayed name", type: "char" },
+                        stage_id: { string: "Stage", type: "many2one", relation: "stage_model" },
+                        json_statustime: { string: "Time per stage", type: "char", default: "{}"},
+                    },
+                    records: [
+                        {
+                            id: 1,
+                            display_name: "first record",
+                            stage_id: 30,
+                            json_statustime: {"10": 604800, "20": 10800},  // 7 days - 3 hours
```

Comment:
no need to quote keys in object (it's kind of confusing here, because those are actually ids)